tasks/async/async.js

```diff
@@ -8,37 +8,55 @@ const {
 /**
  * console.log data got from callbackFunction
  */
-function printDataCallback() {
+function printDataCallback(){
+    let arg = function myFunction(str, data){
+        console.log(data);
+    };
+    callbackFunction(arg);
 }
-
+   
 /**
  * console.log data got from promiseFunction
  */
-function printDataPromise() {
-}
 
+function printDataPromise() {
+    let arg = promiseFunction();
```

Comment:
Actually, this step is not needed (applicable for all functions) but if it is easier for you to read/understand, leave as is.